data/sql/updates/pending_db_world/rev_1719322993021854900.sql

```diff
@@ -0,0 +1,15 @@
+-- Treebole
+UPDATE `creature_template_addon` SET `path_id` = 222150 WHERE (`entry` = 22215);
+
+DELETE FROM `waypoint_data` WHERE `id` = 222150;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`, `move_type`, `action`, `action_chance`, `wpguid`) VALUES
+(222150, 1, 3609.03, 6829.57, 136.69, 5.2105, 300000, 0, 0, 100, 0),
+(222150, 2, 3598.63, 6845.62, 140.703, 2.54014, 0, 0, 0, 100, 0),
+(222150, 3, 3587.26, 6853.75, 141.589, 2.52051, 0, 0, 0, 100, 0),
+(222150, 4, 3568.35, 6864.28, 140.533, 4.41331, 300000, 0, 0, 100, 0),
+(222150, 5, 3587.72, 6853.33, 141.546, 5.70922, 0, 0, 0, 100, 0),
+(222150, 6, 3599.61, 6845.15, 140.649, 5.70922, 0, 0, 0, 100, 0);
+
+DELETE FROM `creature` WHERE (`id1` = 22215);
```

Comment:
in `creature` you delete GUID + id

Reply:
because GUID is unique

Reply:
I usually don't bother when it's a single spawn, but yes, you are right.

Reply:
well double guid and you're fucked

Reply:
what do you mean by double guid?